js_src/_dart_helpers.js

```diff
@@ -1,9 +1,8 @@
 /**
  * react-dart JS interop helpers (used by react_client.dart and react_client/js_interop_helpers.dart)
  */
-function _getProperty(obj, key) { return obj[key]; }
-function _setProperty(obj, key, value) { return obj[key] = value; }
-
+var _reactDartSymbolPrefix = 'react-dart.';
+var _reactDartContextSymbol = Symbol.for(_reactDartSymbolPrefix+'context');
```

Comment:
These could use some doc comments to explain what is happening / why it is necessary.